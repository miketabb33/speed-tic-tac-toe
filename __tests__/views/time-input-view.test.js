import React from 'react'
import renderer from 'react-test-renderer'
import TimeInputView from '../../views/time-input-view'

test('render not hidden', () => {
	const subject = renderer.create(
		<TimeInputView hide = {false} />
	)

	const result = subject.toJSON()
	expect(result).toMatchSnapshot()
})

test('render hidden', () => {
	const subject = renderer.create(
		<TimeInputView hide = {true} />
	)

	const result = subject.toJSON()
	expect(result).toMatchSnapshot()
})