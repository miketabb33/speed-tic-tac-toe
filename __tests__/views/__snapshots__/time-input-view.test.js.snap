// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render hidden 1`] = `
<div
  className="timeInputView clear "
  id="time-input-view"
>
  <div
    className="timeInputText"
  >
    Starting Time:
  </div>
  <input
    className="timeInput"
    defaultValue="10"
    id="time-input"
    maxLength={5}
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    type="text"
  />
</div>
`;

exports[`render not hidden 1`] = `
<div
  className="timeInputView undefined "
  id="time-input-view"
>
  <div
    className="timeInputText"
  >
    Starting Time:
  </div>
  <input
    className="timeInput"
    defaultValue="10"
    id="time-input"
    maxLength={5}
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    type="text"
  />
</div>
`;
